packages/nvlsm/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "nvlsm"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[[package.metadata.build-package.external-files]]
```

Comment:
Don't we need `NVidiaEULAforAWS.pdf` as another external file?

Reply:
Yes, I'll fix in the next revision.